web/styles/compose.css

```diff
@@ -669,22 +669,32 @@ input.recipient_box {
     min-height: var(--compose-recipient-box-min-height);
 }
 
-.tippy-content .compose_control_buttons_container {
-    .compose_gif_icon {
-        bottom: 5px;
-    }
-}
-
 .compose_control_buttons_container {
     margin-right: auto;
     display: flex;
     gap: 4px;
     align-items: center;
 
-    .compose_gif_icon {
+    /* We use the selector in this manner to maintain specificity. */
+    .compose_control_button_container .compose_gif_icon {
         font-size: 22px;
-        height: 18px;
-        line-height: 18px;
+
+        /* Remove top and bottom padding. This is necessary
+         * because `compose_gif_icon` is no longer a flex item.  */
+        padding: 0 5px;
```

Comment:
Hey @karlstolley, you might want to take a look at this solution for addressing the issue where the inner `<a>` element increases the height of `compose_control_button_container`.

Reply:
I think this is fine for now. The compose-box rewrite should help simplify some of this, I hope.